Entities/ServerConfig.cs

```diff
@@ -19,6 +19,235 @@ public class ServerConfig
+		public bool AntispamLinksBeamBan{ get; set; } = false;
+		
+		[Column("antispam_links_imgur")]
+		public bool AntispamLinksImgur{ get; set; } = false;
+		
+		[Column("antispam_links_imgur_ban")]
+		public bool AntispamLinksImgurBan{ get; set; } = false;
+
+		[Column("antispam_tolerance")]
+		public Int64 AntispamTolerance{ get; set; } = 7;
+		
+		[Column("antispam_ignore_members")]
+		public bool AntispamIgnoreMembers{ get; set; } = false;
+		
+		[Column("roleid_operator")]
+		public guid RoleIdOperator{ get; set; } = 0;
+
+		[Column("quickban_duration")]
+		public Int64 QuickbanDuration{ get; set; } = 12;
+
+		[Column("quickban_reason", TypeName = "text")]
+		public string QuickbanReason{ get; set; } = "";
+		
+		[Column("mute_roleid")]
+		public guid MuteRoleId{ get; set; } = 0;
+		
+		[Column("mute_ignore_channelid")]
+		public guid MuteIgnoreChannelId{ get; set; } = 0;
+		
+		[Column("karma_enabled")]
+		public bool KarmaEnabled{ get; set; } = false;
+
+		[Column("karma_limit_mentions")]
+		public Int64 KarmaLimitMentions{ get; set; } = 5;
+
+		[Column("karma_limit_minutes")]
+		public Int64 KarmaLimitMinutes{ get; set; } = 60;
+		
+		[Column("karma_limit_response")]
+		public bool KarmaLimitResponse{ get; set; } = true;
+
+		[Column("karma_currency", TypeName = "varchar(255)")]
+		public string KarmaCurrency{ get; set; } = "cookies";
+
+		[Column("karma_currency_singular", TypeName = "varchar(255)")]
+		public string KarmaCurrencySingular{ get; set; } = "cookie";
+
+		[Column("karma_consume_command", TypeName = "varchar(255)")]
+		public string KarmaConsumeCommand{ get; set; } = "nom";
+
+		[Column("karma_consume_verb", TypeName = "varchar(255)")]
+		public string KarmaConsumeVerb{ get; set; } = "nommed";
+		
+		[Column("log_Channelid")]
+		public guid LogChannelId{ get; set; } = 0;
+		
+		[Column("mod_channelid")]
+		public guid ModChannelId{ get; set; } = 0;
+		
+		[Column("log_bans")]
+		public bool LogBans{ get; set; } = false;
+		
+		[Column("log_promotions")]
+		public bool LogPromotions{ get; set; } = false;
+		
+		[Column("log_deletedmessages")]
+		public bool LogDeletedmessages{ get; set; } = false;
+		
+		[Column("log_editedmessages")]
+		public bool LogEditedmessages{ get; set; } = false;
+		
+		[Column("activity_channelid")]
+		public guid ActivityChannelId{ get; set; } = 0;
+		
+		[Column("log_join")]
+		public bool LogJoin{ get; set; } = false;
+		
+		[Column("log_leave")]
+		public bool LogLeave{ get; set; } = false;
+
+		[Column("log_message_join", TypeName = "text")]
+		public string LogMessageJoin{ get; set; } = "{0} joined the server.";
+
+		[Column("log_message_leave", TypeName = "text")]
+		public string LogMessageLeave{ get; set; } = "{0} left.";
+		
+		[Column("log_mention_join")]
+		public bool LogMentionJoin{ get; set; } = false;
+		
+		[Column("log_mention_leave")]
+		public bool LogMentionLeave{ get; set; } = false;
+		
+		[Column("welcome_pm")]
+		public bool WelcomePm{ get; set; } = false;
```

Comment:
Let's change this to... `WelcomeMessageEnabled` (only the property, leaving the database name as-is)